File: DMOJ/CCC/AttackOfTheCipherTexts.java
// CCC '06 S2 - Attack of the CipherTexts

import java.util.*;
public class AttackOfTheCipherTexts {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		String a = in.nextLine(), b = in.nextLine(), c = in.nextLine();
		for (char t: c.toCharArray()) {
			int index = b.indexOf(t);
			if (index!=-1) {
				System.out.print(a.charAt(index));
			} else {
				System.out.print('.');
			}
		}
		in.close();
	}

}

File: DMOJ/CCC/AttackOfTheCipherTexts.py
# CCC '06 S2 - Attack of the CipherTexts

import sys

input = sys.stdin.readline
a, b, c = input(), input(), input()
for char in c:
    if char == " ":
        print(" ", end="")
    elif char in b:
        print(a[b.index(char)], end="")
    else:
        print(".", end="")
